partiql-tests-validator/src/test/kotlin/org/partiql/tests/validator/PartiQLTestDataValidator.kt

```diff
@@ -301,27 +332,81 @@ class PartiQLTestDataValidator {
     }
 
     @Test
-    fun testMultipleAssertions() {
+    fun testEvaluationSuccessWithEquivalence() {
```

Comment:
nit, for this and others that apply: maybe encoding `Schema` in the name, e.g. `testEvaluationSuccessSchemaWithEquivalence`.

Reply:
Agree with making it clearer that the schema is what's being tested. Changed in https://github.com/partiql/partiql-tests/pull/26/commits/d0988acec5e9ecab6498e03ba7605d03f6ddf517